packages/remix/package.json

```
@@ -28,6 +28,9 @@
         "import": "./build/esm/index.client.js",
         "require": "./build/cjs/index.client.js"
       },
+      "import": {
+        "default": "./build/esm/index.server.js"
+      },
       "node": "./build/cjs/index.server.js"
```

shouldn't this get split into the following?

```suggestion
      "node": {
        "import": "./build/esm/index.server.js",
        "require": "./build/cjs/index.server.js"
      },
```

---

That was my initial change, see the first comment as to why that did not work.